package/gluon-mesh-babel/luasrc/lib/gluon/upgrade/300-gluon-mesh-babel-mkconfig

```diff
@@ -14,7 +15,7 @@ file:write("import-table 254\n")
 file:write("out ip " .. site.next_node.ip6() .. "/128 deny\n")
 file:write("redistribute ip " .. site.next_node.ip6() .. "/128 deny\n")
 file:write("redistribute ip " .. site.prefix6() .. " eq 128  allow\n")
-file:write("redistribute ip " .. site.node_client_prefix6() .. " eq 128  allow\n")
+file:write("redistribute ip " .. l3.node_client_prefix6() .. " eq 128  allow\n")
```

Comment:
I don't know if this is correct at all, since the prefix per doc doesn't seem to be used for routing packets? Or is it being used for that in some kindof crazy way?

Reply:
As far as I understand l3roamd uses this prefix to announce the roaming of a client to another node. The receiving node listens on an address in this range with IP_FREEBIND. To be able to receive such an announcement, nodes between the old and the new node for the client that roamed need to forward these l3roamd packets with this unicast node-client-IP destination address.

Reply:
this is exactly how it works.